### ISSUE TITLE
 In configmgr there is a VPN section

### ISSUE THREAD

**Issue by FranzSchenk** [MicrosoftDocs/windowsserverdocs#3087]
it's like rocket sience!
Why the hell i need to use a powershell script as a package in configmgr to deploy a vpn. In configmgr there is a VPN section. It's not straight forward what ms do.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7453068-95eb-2a42-fd0b-553ae593adb0
* Version Independent ID: b39b2cb7-25b1-2af3-4954-dbcc04c836e9
* Content: [Configure Windows 10 Client Always On VPN Connections](https://docs.microsoft.com/en-us/windows-server/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections#bkmk_fullscript)
* Content Source: [WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections.md)
* Product: **windows-server-threshold**
* Technology: **networking-ras**
* GitHub Login: @shortpatti
* Microsoft Alias: **pashort**

**Comment by AndyChips** [comment]
Yes, I agree. Why use this incredibly unpleasant deployment method when SCCM already seems to have a function built-in. I see little documentation regarding this as well.
